hudi-utilities/src/main/java/org/apache/hudi/utilities/HoodieMetadataTableValidator.java

```diff
@@ -399,7 +400,9 @@ public void doMetadataTableValidation() {
     Set<String> baseFilesForCleaning = Collections.emptySet();
 
     // check metadata table is available to read.
-    checkMetadataTableIsAvailable();
+    if (!checkMetadataTableIsAvailable()) {
+      return;
+    }
```

Comment:
even if Metadata table does not have any commits, do we need to add an else block here and check that there are no completed commits in data table?

Reply:
Sure, changed. add another else block in checkMetadataTableIsAvailable()
If there is no completed commits neither in data table nor MDT, HoodieMetadataTableValidator will skip current loop and log an info message.